server/services/forms/schemas/update-correspondent-details.js

```diff
@@ -7,6 +7,13 @@ module.exports = async ({ caseId, stageId, context, user, requestId }) => {
     const { data: { address, ...data } } = await caseworkService.get(`/case/${caseId}/correspondent/${context}`, { headers: { ...User.createHeaders(user), 'X-Correlation-Id': requestId } });
     return Form()
         .withTitle('Edit correspondent details')
+        .withField(
+            Component('dropdown', 'type')
+                .withValidator('required', 'The correspondent must have a type')
+                .withProp('label', 'Correspondent type')
+                .withProp('choices', 'SELECTABLE_CORRESPONDENT_TYPES')
```

Comment:
It would be good to create a ticket to further look at this again in the future, what if we need case type specific types?